doc/quick.md

```diff
@@ -62,49 +52,90 @@ These steps have been tested on:
     HugePages_Surp:        0
 
     ```
+    
+    If fewer than 512 pages are available, the page count should be configured as necessary, accounting for any other co-resident workloads which may require them. e.g.
+
+    ```bash
+    echo 512 | sudo tee /sys/kernel/mm/hugepages/hugepages-2048kB/nr_hugepages
+    ```
+
+    This change can be made persistent across reboots by adding the required value to `/etc/sysctl.conf`, thus:
+    ```
+    vm.nr_hugepages = 512
+    ```
+
+    > If you modify the huge page configuration of a node, you *must* restart the kubelet or reboot the node
+    
+
+2.  Load the NBD and XFS kernel modules (if not already loaded).
```

Comment:
This step is not mandatory actually. Only if you want to use nbd, nbd should be loaded and -- XFS is optional but i recommond using XFS. Should make this step optinal?

Reply:
If loading XFS is optional but recommended, can I ask what the basis of that recommendation is?  What's the potential benefit of loading it and does it apply only in specific circumstances?